lib/active_model/serializer/reflection.rb

```diff
@@ -113,8 +173,8 @@ def build_association(parent_serializer, parent_serializer_options, include_slic
         association_value = value(parent_serializer, include_slice)
         serializer_class = parent_serializer.class.serializer_for(association_value, reflection_options)
         reflection_options[:include_data] = include_data?(include_slice)
-        reflection_options[:links] = @_links
-        reflection_options[:meta] = @_meta
+        reflection_options[:links] = options[:links]
+        reflection_options[:meta] = options[:meta]
```

Comment:
Removing dependency on instance variables makes it possible to just use reflection_options instead of the explicit passing of @_links and @_meta